Sources/AWSLambdaEvents/Utils/DateWrappers.swift

```diff
@@ -67,3 +69,30 @@ public struct ISO8601WithFractionalSecondsCoding: Decodable {
         return formatter
     }
 }
+
+@propertyWrapper
+public struct DateTimeCoding: Decodable {
```

Comment:
I'm unsure about the naming since it is very generic? Is there a special name for this format? @tomerd @weissi

Reply:
I agree that we need to find a more specific name. Is this used only in SES?

Reply:
lets also add a unit test for this

Reply:
> I'm unsure about the naming since it is very generic? Is there a special name for this format? @tomerd @weissi 

RFC is particularly unhelpful https://tools.ietf.org/html/rfc5322#section-3.3

Reply:
It’s appears to be a standard internet format. It is the same one used in http headers.

Reply:
AWS docs mention 5322: https://docs.aws.amazon.com/ses/latest/DeveloperGuide/sending-concepts-email-format.html so perhaps we name it `RFC5322DateTimeCoding`?

Reply:
renamed